.devcontainer/devcontainer.json

```diff
@@ -12,13 +12,13 @@
 
   // Use this environment variable if you need to bind mount your local source code into a new container.
   "remoteEnv": {
-    "LOCAL_WORKSPACE_FOLDER": "${localWorkspaceFolder}"
+    "LOCAL_WORKSPACE_FOLDER": "${localWorkspaceFolder:-.}"
   },
 
   // Use 'forwardPorts' to make a list of ports inside the container available locally.
-  "forwardPorts": [3000, 8000, 8080]
+  "forwardPorts": [3000, 8000, 8080],
 
   // Use 'postCreateCommand' to run commands after the container is created.
   // Note: Useful for when they open it and want everything to just start
-  // "postCreateCommand": "docker compose up -d"
+  "postCreateCommand": "bash ./.devcontainer/setup.sh"
 }
```

Comment:
If you don't want running bash, you can do chmod 755 on the shell script, so it becomes an executable